STYLEGUIDE.md

```diff
@@ -72,3 +78,74 @@ import 'camera.dart';
 import 'game.dart';
 import 'viewport.dart';
 ```
+
+### Identifiers
+
+Use all [effective dart patterns](https://dart.dev/guides/language/effective-dart)
+
+* Avoid using global constants, but if you do use, name it prefixing with a `k`.
```

Comment:
Interesting, never knew about the `k` thing

Reply:
Fun fact: thats the origin of kPop.

Reply:
I don't quite agree with this rule, I think we should either ban it altogether or allow. prefixing things with k is weird

Reply:
@luanpotter the `k` thing is how Flutter does on its source base, do I think this rule is fine.

Reply:
yeah but I think Flutter has reasons to need global variables, I don't think audioplayers should ever pollute the global scope like that.

Reply:
I mean flame haha

Reply:
if we ever want to add a global constant for some reason we can revisit this rule, I see no reason to encourage this behaviour now